src/shared/styles.ts

```diff
@@ -24,7 +15,318 @@ export const GlobalStyle = createGlobalStyle`
+  },
+  green: {
+    50: "#e8f5e9",
+    100: "#c8e6c9",
+    200: "#a5d6a7",
+    300: "#81c784",
+    400: "#66bb6a",
+    500: "#4caf50",
+    600: "#43a047",
+    700: "#388e3c",
+    800: "#2e7d32",
+    900: "#1b5e20",
+    a100: "#b9f6ca",
+    a200: "#69f0ae",
+    a400: "#00e676",
+    a700: "#00c853",
+  },
+  lightgreen: {
+    50: "#f1f8e9",
+    100: "#dcedc8",
+    200: "#c5e1a5",
+    300: "#aed581",
+    400: "#9ccc65",
+    500: "#8bc34a",
+    600: "#7cb342",
+    700: "#689f38",
+    800: "#558b2f",
+    900: "#33691e",
+    a100: "#ccff90",
+    a200: "#b2ff59",
+    a400: "#76ff03",
+    a700: "#64dd17",
+  },
+  lime: {
+    50: "#f9fbe7",
+    100: "#f0f4c3",
+    200: "#e6ee9c",
+    300: "#dce775",
+    400: "#d4e157",
+    500: "#cddc39",
+    600: "#c0ca33",
+    700: "#afb42b",
+    800: "#9e9d24",
+    900: "#827717",
+    a100: "#f4ff81",
+    a200: "#eeff41",
+    a400: "#c6ff00",
+    a700: "#aeea00",
+  },
+  yellow: {
+    50: "#fffde7",
+    100: "#fff9c4",
+    200: "#fff59d",
+    300: "#fff176",
+    400: "#ffee58",
+    500: "#ffeb3b",
+    600: "#fdd835",
+    700: "#fbc02d",
+    800: "#f9a825",
+    900: "#f57f17",
+    a100: "#ffff8d",
+    a200: "#ffff00",
+    a400: "#ffea00",
+    a700: "#ffd600",
+  },
+  amber: {
+    50: "#fff8e1",
+    100: "#ffecb3",
+    200: "#ffe082",
+    300: "#ffd54f",
+    400: "#ffca28",
+    500: "#ffc107",
+    600: "#ffb300",
+    700: "#ffa000",
+    800: "#ff8f00",
+    900: "#ff6f00",
+    a100: "#ffe57f",
+    a200: "#ffd740",
+    a400: "#ffc400",
+    a700: "#ffab00",
+  },
+  orange: {
+    50: "#fff3e0",
+    100: "#ffe0b2",
+    200: "#ffcc80",
+    300: "#ffb74d",
+    400: "#ffa726",
+    500: "#ff9800",
+    600: "#fb8c00",
+    700: "#f57c00",
+    800: "#ef6c00",
+    900: "#e65100",
+    a100: "#ffd180",
+    a200: "#ffab40",
+    a400: "#ff9100",
+    a700: "#ff6d00",
+  },
+  deeporange: {
+    50: "#fbe9e7",
+    100: "#ffccbc",
+    200: "#ffab91",
+    300: "#ff8a65",
+    400: "#ff7043",
+    500: "#ff5722",
+    600: "#f4511e",
+    700: "#e64a19",
+    800: "#d84315",
+    900: "#bf360c",
+    a100: "#ff9e80",
+    a200: "#ff6e40",
+    a400: "#ff3d00",
+    a700: "#dd2c00",
+  },
+  brown: {
+    50: "#efebe9",
+    100: "#d7ccc8",
+    200: "#bcaaa4",
+    300: "#a1887f",
+    400: "#8d6e63",
+    500: "#795548",
+    600: "#6d4c41",
+    700: "#5d4037",
+    800: "#4e342e",
+    900: "#3e2723",
+  },
+  grey: {
+    50: "#fafafa",
+    100: "#f5f5f5",
+    200: "#eeeeee",
+    300: "#e0e0e0",
+    400: "#bdbdbd",
+    500: "#9e9e9e",
+    600: "#757575",
+    700: "#616161",
+    800: "#424242",
+    900: "#212121",
+  },
+  bluegrey: {
+    50: "#eceff1",
+    100: "#cfd8dc",
+    200: "#b0bec5",
+    300: "#90a4ae",
+    400: "#78909c",
+    500: "#607d8b",
+    600: "#546e7a",
+    700: "#455a64",
+    800: "#37474f",
+    900: "#263238",
+  },
+};
+
+/**
+ * Used for theming components and amplify
+ */
+export const colors: Colors = {
```

Comment:
can we have that on top of the file?
Since the color palette is really big, I think it's better to have it on top.